synapseclient/client.py

```diff
@@ -2079,12 +1993,14 @@ def _getRawAnnotations(self, entity, version=None):
             uri = f"/entity/{id_of(entity)}/annotations2"
         return self.restGET(uri)
 
-    @deprecated.sphinx.deprecated(
+    @deprecated(
         version="2.1.0",
-        reason="deprecated and replaced with :py:meth:`get_annotations`",
+        reason="deprecated and replaced with `get_annotations`",
     )
-    def getAnnotations(self, entity, version=None):
-        """deprecated and replaced with :py:meth:`get_annotations`"""
+    def getAnnotations(
```

Comment:
Unrelated to this PR, but do you think it'd be worthwhile to remove these since they've been deprecated for some time? 

It can be another ticket.

Reply:
Yeah we can go ahead and remove these I'm the 4.0 release. Would you like to create the ticket, otherwise I will

Reply:
Done: https://sagebionetworks.jira.com/browse/SYNPY-1392